buildspec.yml

```diff
@@ -14,6 +14,10 @@ phases:
     commands:
       - echo Testing the newly built Docker image...
       - docker run --name test -d -e RAILS_ENV=test $CONTAINER_NAME:test /bin/bash -c "tail -f /dev/null"
+      - git config --global credential.helper store
+      - echo "https://$GIT_USERNAME:$GIT_PASSWORD@github.com" >
+        /root/.git-credentials
+      - ./test.sh -i $INFRASTRUCTURE_NAME
```

Comment:
you don't need this line. that the dalmatian test suite

Reply:
line 20?

Reply:
I've dd line 20

Reply:
Could you please clarify what you meant by 'although given you run your tests in docker you might need to do more.'? 
Do you mean we need to run more tests in docker? If yes, which type of tests do you suggest?

Reply:
the container wont know about changes made outside and indeed i think this might be needed in the container itself.